app/tests/unit/components/stackpole.spec.js

```diff
@@ -0,0 +1,96 @@
+const helper = require('../../common/helper');
+const stackpole = require('../../../src/components/stackpole');
+
+helper.logHelper();
+
+describe('register', () => {
+  const mockExecuteFn = jest.fn();
+  const mockXformFn = jest.fn();
+
+  afterEach(() => {
+    mockExecuteFn.mockClear();
+    mockXformFn.mockClear();
+  });
+
+  it('should register a method and call it with a transform', () => {
+    stackpole.register('testOne', mockExecuteFn, mockXformFn);
+
+    stackpole.testOne();
+
+    expect(mockExecuteFn).toHaveBeenCalledTimes(1);
+    expect(mockXformFn).toHaveBeenCalledTimes(1);
+
+  });
+
+  it('should register a method and call it without a transform', () => {
+    stackpole.register('testTwo', mockExecuteFn);
+
+    stackpole.testTwo();
+
+    expect(mockExecuteFn).toHaveBeenCalledTimes(1);
+    expect(mockXformFn).toHaveBeenCalledTimes(0);
+
+  });
+
+  it('should register a method and do nothing if not functions', () => {
+    stackpole.register('testThree', 'not an execute function', 'this is not valid');
+
+    stackpole.testThree();
+
+    expect(mockExecuteFn).toHaveBeenCalledTimes(0);
+    expect(mockXformFn).toHaveBeenCalledTimes(0);
+
+  });
+
+  it('should register a method and call it, ignoring invalid transform', () => {
+    stackpole.register('testFour', mockExecuteFn, 'this is not valid');
+
+    stackpole.testFour();
+
+    expect(mockExecuteFn).toHaveBeenCalledTimes(1);
+    expect(mockXformFn).toHaveBeenCalledTimes(0);
+
+  });
+
+  it('should register a method and call it and call multiple executors', () => {
+    stackpole.register('testFive', [mockExecuteFn, mockExecuteFn, mockExecuteFn], mockXformFn);
+
+    stackpole.testFive();
+
+    expect(mockExecuteFn).toHaveBeenCalledTimes(3);
+    expect(mockXformFn).toHaveBeenCalledTimes(1);
+
+  });
+
+  it('should register a method and call it gracefully when executors error', () => {
+    stackpole.register('testSix', jest.fn().mockImplementation(() => {throw new Error('bad');}), mockXformFn);
```

Comment:
Just for future reference, `jest.fn(() => {...})` is equivalent to the explicit mockImplementation function.